src/util/log.d

```diff
@@ -215,27 +215,47 @@ struct EventInfo
     size_t line;
 }
 
+auto fileLogger(string name, uint levels = LogLevel.info.orAbove)
+{
+    return fileLogger(new Layout, name, levels);
```

Comment:
return new FileLogger!Layout(new Layout, name, levels);

Just a convenience function for the simple case of using the standard layout.
If you do more complicated stuff, you have no convenience function but a good example.

So: remove

    auto fileLogger(Layout)
        (Layout layout, string name, uint levels = LogLevel.info.orAbove)